data/language/en-GB.yml

```diff
@@ -2334,3 +2334,4 @@ strings:
   2279: "{COLOUR BLACK}Checksum"
   2280: "Play sound effects for new messages"
   2281: "{SMALLFONT}{COLOUR BLACK}When enabled, new messages will be accompanied by a sound effect of an appropriately cheering crowd."
+  2282: "{SMALLFONT}{COLOUR BLACK}Reverse the route order table"
```

Comment:
is it called order table in game?

Reply:
I hesitated to call it such as well. The tab is actually called 'Routing', but the command tooltips refer to 'orders'. As a compromise, I thought I'd refer to both in this way.